articles/howto-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,281 @@
+
+Create a counter.js file in the pages folder and then add this code snippet.
+```JavaScript
+//counter code
+import React from "react"
+
+class Counter extends React.Component {
+    constructor() {
+        super()
+        this.state = { count: 0 }
+    }
+    render() {
+        return <div style={{ color: 'blue' }}>
+            <h1>Counter</h1>
+            <p>current counter: {this.state.count}</p>
+            //plus button
+            <button onClick={() => this.setState({ count: this.state.count + 1 })} style={{ color: 'tomato' }}>PLUS</button>
+            //minus button
+            <button onClick={() => this.setState({ count: this.state.count - 1 })} style={{ color: 'tomato' }}>MINUS</button>
+            <br></br>
+            <br></br>
+            <br></br>
+            // To go back to the homepage
+            <Link to="/">Back</Link>
+        </div>
+    }
+}
+
+export default
+```
+
+### Using React Components
+
+Using react components to build your static site will help you get the job done faster. With react you can make parts of your code as a component, component codes are reusable and can be placed in any of your react code file to be used again.
+
+Yet, I highly recommend you check out the [react component documentation](https://reactjs.org/docs/react-component.html) to gain a better understanding of react components.
+
+The counter code in your counter.js file is already a component because it extends the react component class. You will use it in the index.js file but as a component,
+by adding this piece of code.
+
+
+```bash
+<Counter></Counter>
+```
+
+Also, import the counter component by placing this code at the top:
+
+`bash
+import { Link } from "gatsby"
+`
+
+### Using plugins in GatsbyJS
+
+What if you want to extend your site to have cool features that you don’t want to build from the scratch, GatsbyJS offers [plugins](https://www.gatsbyjs.com/plugins/) built by a community of developers who want to share cool and awesome features with everyone.
+
+We will be using a plugin for [Typogyaphy Js](https://kyleamathews.github.io/typography.js/) (a CSS framework) to style up the CSS by default.
+
+To use this plugin, you have to do this:
+
+```bash
+npm install gatsby-plugin-typography react-typography typography
+```
+
+When the installation is done go to the *gatsby-config.js* file, you want to tell Gatsby that you want to use this plugin.
+
+Add these lines of code in the *gatsby-config.js* file.
+
+``` JavaScript
+//plugin code
+module.exports = {
+plugins: [ `gatsby-plugin-typography`]
+}
+```
+
+### Building the site on a web server
+
+So far so good, your site has been running on your local server which is `http://localhost:8000`, what’s the use of building a static site without showing it to your friends, family, and probably your boss online.
```

Comment:
- Remove "So far so good"
- Add comma after "local server"
- "Remove "what's the use of..." sentence = inappropriate (doesn't match tone of technical article)

Reply:
Done